cmd/swarm/swarm-smoke/upload_and_sync.go

```diff
@@ -50,11 +56,77 @@ func uploadAndSyncCmd(ctx *cli.Context, tuid string) error {
 		metrics.GetOrRegisterCounter(fmt.Sprintf("%s.timeout", commandName), nil).Inc(1)
 
 		// trigger debug functionality on randomBytes
+		err := triggerChunkDebug(randomBytes)
+		if err != nil {
+			log.Error("test timed out and triggerChunkDebug also produced error", "err", err)
```

Comment:
you don't really need to print this error. if you would just create a new error and wrap it with the timeout error - the cli util will print this. so replace with
```
e := fmt.Errorf("timeout after %v sec", timeout)
err := triggerChunkDebug(randomBytes)
if err != nil {
      e = fmt.Errorf("%v; triggerChunkDebug failed: %v",e, err)
}

return e
```

Or something along those lines